src/pyskindose/format_export_data.py

```diff
@@ -0,0 +1,475 @@
+                "orientation": self.Patient["orientation"],
+                "offsets": self.PatientOffsets,
+            },
+            "table": {
+                "table_surface": {
+                    "x": self.Table.r[:, 0].tolist(),
+                    "y": self.Table.r[:, 1].tolist(),
+                    "z": self.Table.r[:, 2].tolist(),
+                },
+                "triangle_vertex_indices": {
+                    "i": self.Table.ijk[:, 0].tolist(),
+                    "j": self.Table.ijk[:, 1].tolist(),
+                    "k": self.Table.ijk[:, 2].tolist(),
+                },
+                "table_length": self.Table.table_length,
+            },
+            "pad": {
+                "pad_surface": {
+                    "x": self.Pad.r[:, 0].tolist(),
+                    "y": self.Pad.r[:, 1].tolist(),
+                    "z": self.Pad.r[:, 2].tolist(),
+                },
+                "triangle_vertex_indices": {
+                    "i": self.Pad.ijk[:, 0].tolist(),
+                    "j": self.Pad.ijk[:, 1].tolist(),
+                    "k": self.Pad.ijk[:, 2].tolist(),
+                },
+            },
+            "dose_map": [(ind, dose) for ind, dose in enumerate(self.DoseMap.tolist()) if dose > 0.0],
+            "corrections": {
+                "correction_value_index": self.Hits,
+                "backscatter": self.BackscatterCorrection,
+                "medium": self.MediumCorrection,
+                "table": self.TableCorrection,
+                "inverse_square_law": self.InverseSquareLawCorrection,
+            },
+            "events": self.Events.to_dict(),
+        }
+
+    def to_json(self) -> str:
+        """Converts the output data into a JSON string
+
+        Returns
+        -------
+        str
+            A JSON formatted string containing the output data
+        """
+        return json.dumps(self.to_dict())
+
+
+def format_analysis_result_for_export(
+    analysis_result: Dict[str, Any],
+    data_norm: pd.DataFrame,
+    patient: Phantom,
+    table: Phantom,
+    pad: Phantom,
+    settings: PyskindoseSettings,
+) -> Union[PySkinDoseOutput, dict[str, Any], str]:
+    """Formats the result of the PySkinDose analysis into a PySkinDoseOutput class instance that has a methods for
+    converting the result to either a dict or a JSON string to facilitate building custom visualizations and for other
+    custom implementations of the PySkinDose calculated data.
+
+    Parameters
+    ----------
+    analysis_result : dict[str, Any]
+        The dict resulting from the call to pyskindose.calculate_dose.calculate_dose.calculate_dose
+    data_norm : pd.DataFrame
+        The RDSR data, normalized for compliance with PySkinDose's use of units etc.
+    patient : Phantom
+        An instance of the Phantom class that represents the patient
+    table : Phantom
+        An instance of the Phantom class that represents the treatment table
```

Comment:
shouldnt it be "A phantom that represents the treatment table"? 

It is clear from the "patient: Phantom" that the parameter table is of instance Phantom

Reply:
It can be considered clear that it is an instance of the phantom class from that but I believe it is more clear to read the text when we specify it this way. I think I would be more confused of reading the sentence when specifying the table as a phantom. But I'll leave i t up to you to decide :)